app/controllers/comments_controller.rb

```diff
@@ -1,30 +1,51 @@
 class CommentsController < ApplicationController
+  load_and_authorize_resource
```

Comment:
- [ ] Good job using cancancan for authorization, however, you are to use it for only post authorization, so kindly use it in your post controller alone.

Reply:
Hello, @Ghiftee. Thanks for the review. Why should only use it in the post controller? I also have role authorizations in my comments_counter.

Reply:
Hi @XsidX sorry about that, my mistake, please submit it again and it would be reviewed. Thank you

Reply:
Thank you @Ghiftee .